### ISSUE TITLE
Support reading channels from somewhere other than this.$options for Vue Class syntax

### ISSUE THREAD

**Issue by RealRobin-com** [mclintprojects/actioncable-vue#25]
**Is your feature request related to a problem? Please describe.**
I would like to be able to define my channels and callbacks within my class body instead of the @Component decorator from [Vue Class Component](https://class-component.vuejs.org/)

**Describe the solution you'd like**
a sentinel value like `channels` read from a class level method instead of `this.$options`

**Describe alternatives you've considered**
expose a public api to manually run the logic in the mixin so it doesn't matter when the channels are stored

**Additional context**
# This is what I have to do today.
```typescript

@Component({
  channels: {
    ConversationChannel: {
      connected() {
        console.log("connected");
      },
      rejected() {
        console.log("rejected");
      },
      received(data) {
        console.log(`received ${JSON.stringify(data)}`);
      },
      disconnected() {
        console.log("disconnected");
      }
    }
  }
})
export default class ConversationComponent extends Vue {
  @Prop() private id!: string;

  async mounted() {
    this.$cable.subscribe({
      channel: "ConversationChannel",
      id: this.id
    });
  }
}
```

# Requested usage
```typescript

@Component
export default class ConversationComponent extends Vue {
  @Prop() private id!: string;

  async mounted() {
    this.$cable.subscribe({
      channel: "ConversationChannel",
      id: this.id
    });
  }

  get channels() {
    return {
      ConversationChannel: {
        connected() {
          console.log("connected");
        },
        rejected() {
          console.log("rejected");
        },
        received(data) {
          console.log(`received ${JSON.stringify(data)}`);
        },
        disconnected() {
          console.log("disconnected");
        }
      }
    };
  }
}
```

**Comment by mclintprojects** [comment]
@RealRobin-com I just published v2.2.0 that adds your feature. Kindly update and let me know how it goes.

**Comment by RealRobin-com** [comment]
Looks good and works great. Thanks for the extremely quick feature.
